trunk/includes/pardot-api-class.php

```diff
@@ -565,21 +565,28 @@ function get_response( $item_type, $args = array(), $property = 'result', $paged
         else if ($this->auth_type == 'sso') {
             $headers = array(
                 'Authorization' => 'Bearer ' . $this->api_key,
-                'Pardot-Business-Unit-Id' => $this->business_unit_id,
-                'offset' => $paged > 1 ? ($paged - 1) * 200 : 0
+                'Pardot-Business-Unit-Id' => $this->business_unit_id
             );
 
+		    $body = array_merge( $args,
```

Comment:
IMO, we don't need to merge `$args` here. According to the phpdoc, `$args` is meant to be used for query string parameters. This function doesn't really have a way to accept values for the request body.

Merging `$args` probably won't break anything for now, but it might break something in the future.